src/cms/templates/pages/page_tree.html

```diff
@@ -86,11 +86,14 @@ <h1 class="heading">{% trans 'Pages' %}</h1>
             {% get_last_root_page pages as last_root_page %}
             {% recursetree pages %}
                 {% get_translation node language.slug as page_translation %}
+                {% get_children_in node pages as children_in_filter %}
+                {% get_depth_in node pages as page_depth %}
+                {% is_shown_by_root node last_root_page pages as is_shown %}
                 {% include "pages/page_tree_node.html" with page=node %}
                 {% if not node.is_leaf_node %}
                     {{children}}
                 {% endif %}
-                {% if node == last_root_page %}
+                {% if node == last_root_page or not is_shown %}
                     <tr data-drop-id="{{ node.id }}" data-drop-position="right" class="drop drop-between h-3 hidden" title="test"><td colspan="9"><div><span></span></div></td></tr>
                 {% endif %}
```

Comment:
These lines have the purpose to add a drop region after the last root page.
As drag&drop is disabled, when the page filter feature is active, we won't need this template tag here for now.